sklearn/linear_model/_bayes.py

```diff
@@ -394,12 +412,12 @@ def _update_coef_(
                 [X.T, U / (eigen_vals_ + lambda_ / alpha_)[None, :], U.T, y]
             )
 
-        rmse_ = np.sum((y - np.dot(X, coef_)) ** 2)
+        mse_ = np.sum((y - np.dot(X, coef_)) ** 2)
```

Comment:
Don't we need to compute the weighted mse here as well?

Reply:
It's computing the weighted mse because the `X `and `y` are scaled by `sample_weight` in `_rescale_data`